pkg/apiserver/user/sqlauth/sqlauth.go

```diff
@@ -9,7 +13,36 @@ import (
 	"github.com/pingcap/tidb-dashboard/pkg/tidb"
 )
 
-const typeID utils.AuthType = 0
+const (
+	typeID utils.AuthType = 0
+
+	// Base privileges
+	privAll             string = "ALL PRIVILEGES"
+	privProcess         string = "PROCESS"
+	privShowDB          string = "SHOW DATABASES"
+	privSystemVarAdmin  string = "SYSTEM_VARIABLES_ADMIN"
+	privSuper           string = "SUPER"
+	privConfig          string = "CONFIG"
+	privDashboardClient string = "DASHBOARD_CLIENT"
+	// Extra privileges when TiDB SEM is enabled
+	privRestrictedTablesAdmin string = "RESTRICTED_TABLES_ADMIN"
+	privRestrictedStatusAdmin string = "RESTRICTED_STATUS_ADMIN"
+)
+
+// TiDB config response
+//
+// "security": {
+//   ...
+//   "enable-sem": true/false,
+//   ...
+// },
+type tidbSecurityConfig struct {
+	Security tidbSEMConfig `json:"security"`
+}
+
+type tidbSEMConfig struct {
+	EnableSem bool `json:"enable-sem"`
```

Comment:
SEM enable status should be read from TiDB

Reply:
@crazycs520 told me we can get it from TiDB `/config` HTTP API.

Reply:
Yes, we should get it from TiDB `config` HTTP API.

Reply:
According to Golang's naming convention it should `EnableSEM`